src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -29,7 +59,38 @@ public function __construct(
     protected function configure(): void
     {
         $this
-            ->setHelp('Anonymize database')
+            ->setHelp(<<<TXT
+            Anonymize a given backup file or the current database.
+
+            This command will successively perfom these steps:
+                1/ Backup current database,
+                2/ Restore of the given backup file,
+                3/ Anonymize the database,
+                4/ Backup of the anonymized database overwritting the given backup file,
+                5/ Restore backup file from step 1.
+
+                If used with --current-database option, step 2 is skipped.
+                If used with with --no-restore option, step 1 and 4 are skipped.
+            TXT)
```

Comment:
A few proposals of corrections, rewordings and formatting:

```suggestion
            ->setHelp(
                <<<TXT
                Anonymize a given backup file or the current database.

                This command will successively perform these steps:
                    1/ Backup the current database,
                    2/ Restore the given backup file,
                    3/ Anonymize the database,
                    4/ Backup the anonymized database by overwriting the given backup file,
                    5/ Restore the backup file produced at step 1.

                If called with the --current-database option, step 2 is skipped.
                If called with the --no-restore option, step 1 and 4 are skipped.
                TXT
            )
```

I think I would have used "local" instead of "current" as well.